src/python/bot/minimizer/js_minimizer.py

```diff
@@ -22,6 +22,13 @@
 from . import utils
 
 
+def step_back_while(cur_index, condition):
+  """Helper function. Decreases index from cur until condition is satisfied."""
```

Comment:
nit: "**while** condition is satisfied"? "Until" has an opposite logic, I believe. "as long as condition is satisfied" could also work